src/frontend/web_application/src/components/InputFileGroup/index.jsx

```diff
@@ -116,8 +117,8 @@ class InputFileGroup extends Component {
         {descr && <p>{descr}</p>}
 
         {this.state.files.length > 0 ? (
-          this.state.files.map((file) => (
-            <File file={file} onRemove={this.resetForm} />
+          this.state.files.map((file, i) => (
+            <File key={i} file={file} onRemove={this.resetForm} />
```

Comment:
No array index in key… Mais que fait `eslint` ? :p

Reply:
at least index is better than nothing :)